### PR TITLE
Cell

### PR DESCRIPTION
**Pull request by mikez321**
All tests for cell class have been created based on Pry workflow.  Combined a few early similar tests.  A test now exists for optional showing "S" in ship locations.

### CODE REVIEW

File: test/cell_test.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    cell = Cell.new("B4")
+
+    assert_instance_of Cell, cell
+  end
+
+  def test_it_starts_with_a_coordinate_and_empty
+    cell = Cell.new("B4")
+
+    assert_equal "B4", cell.coordinate
+    assert_nil = cell.ship
```
Comment (Mariana-21) [inline test/cell_test.rb:18]: Not sure I understand why you're using assert_nill maybe we can talk about that

Comment (mikez321) [reply test/cell_test.rb:18]: image

The pry workflow has it in there as wanting it to return nil.  I thought it was weird too but it should work...

Comment (mikez321) [reply test/cell_test.rb:18]: And I know it's not called "pry workflow" but I can't ever remember what the word is....

File: test/cell_test.rb [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  def test_it_renders_an_x_if_a_ship_is_sunk
+    cell_2 = Cell.new("C3")
+    cruiser = Ship.new("Cruiser", 3)
+
+    cell_2.place_ship(cruiser)
+    assert_equal false, cruiser.sunk?
+
+    cell_2.fire_upon
+    2.times { cruiser.hit }
```
Comment (Mariana-21) [inline test/cell_test.rb:101]: Not sure I understand the this line (101)

**Review by Mariana-21 — COMMENTED** [review]
Everything looks to be testing accurately, some refactoring could be done but just small things overall nice test!

**Comment by mikez321** [comment]
2 times do |cruiser.hit|
end

It was just a different way of writing out cruiser.hit twice.  Where do you think it could be refactored?  I'm happy to adjust tests if ya like.

**Pull request merged**